Vostok.Logging.File/FileLog.cs

```diff
@@ -116,9 +131,22 @@ internal FileLog(IMultiFileMuxer muxer, Func<FileLogSettings> settingsProvider)
         public static void FlushAll() => FlushAllAsync().GetAwaiter().GetResult();
 
         /// <summary>
-        /// The number of events dropped by this <see cref="FileLog"/> instance due to events queue overflow.
+        /// Refreshes all caches from all FileLog instances asynchronously. Throws in case of error in any of them.
         /// </summary>
-        public long EventsLost => eventsLost;
+        /// <exception cref="FileLogException"></exception> // TODO: make sure what exceptions are possible and list them here.
+        public static Task RefreshAllSettingsAsync() => Task.WhenAll(
+            Instances
+               .Select(
+                    x => x.TryGetTarget(out var target)
+                        ? target.RefreshSettingsAsync()
```

Comment:
Just in case: let's make sure that calling `RefreshSettingsAsync` on a disposed `FileLog` instance will not result in any weird side effects such as file reopening or infinite waiting.